# HG changeset patch
# User Rik <rik@octave.org>
# Date 1373397753 25200
#      Tue Jul 09 12:22:33 2013 -0700
# Node ID c55df4e5e216d24d06428b26e5339cbbeb74abb0
# Parent  bf4c95a3bba2f118166efe16fceaea0346fcbc9e
doc: Fix missing @deftypefnx entry in delaunay.m.

* scripts/geometry/delaunay.m: Add missing @deftypefnx entry.

diff --git a/scripts/geometry/delaunay.m b/scripts/geometry/delaunay.m
--- a/scripts/geometry/delaunay.m
+++ b/scripts/geometry/delaunay.m
@@ -13,17 +13,17 @@
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {} delaunay (@var{x}, @var{y})
-## @deftypefn  {Function File} {} delaunay (@var{x})
+## @deftypefnx {Function File} {} delaunay (@var{x})
 ## @deftypefnx {Function File} {} delaunay (@dots{}, @var{options})
 ## @deftypefnx {Function File} {@var{tri} =} delaunay (@dots{})
 ## Compute the Delaunay triangulation for a 2-D set of points.
 ## The return value @var{tri} is a set of triangles which satisfies the
 ## Delaunay circum-circle criterion, i.e., only a single data point from
 ## [@var{x}, @var{y}] is within the circum-circle of the defining triangle.
 ## The input @var{x} may also be a matrix with two columns where the first
 ## column contains x-data and the second y-data.
